azure-pipelines.yml

```diff
@@ -67,11 +67,18 @@ jobs:
 
      ls -Recurse -Include global.json | %{
        $v = $_ | Get-Content -Raw | ConvertFrom-Json | %{ if ($_.tools.dotnet) {$_.tools.dotnet} else {$_.sdk.version} }
-       if ($v -and (-not $v.EndsWith('-*'))) {
-        Write-Host "Installing .NET CLI version $v from $_"
-        & $installScript -Version $v -InstallDir $installDir
+       $isSensibleVersion = $v.Split('.')[0] -as [int] -le 6
```

Comment:
"6" seems a bit short sighted. That just means in a year it's going to explode and no one will know why. :-)  Can it be something like "30"? That should buy us a few decades (and still skip the 42.42.42.42 devs like to use)

Reply:
Any number we place here is half arbitrary and a heuristic. So sure, I can only imagine 9.9.9... being used, but doesn't feel likely.

Reply:
It's just that in 2022, we'll be on .NET 7, which will render everything higher than 6.  If people are using realistic numbers as fake numbers, we are in big trouble.